src/fsharp/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj

```diff
@@ -17,8 +17,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <FsYaccOutputFolder>$(IntermediateOutputPath)$(TargetFramework)/</FsYaccOutputFolder>
-    <FsLexOutputFolder>$(IntermediateOutputPath)$(TargetFramework)/</FsLexOutputFolder>
+    <FsYaccOutputFolder>$(IntermediateOutputPath)$(TargetFramework)\</FsYaccOutputFolder>
```

Comment:
do these slashes need to change? they aren't valid on non-Windows OS's.

Reply:
@baronfel --- yep because sourcelink test doesn't match otherwise.